examples/translate.yaml

```diff
@@ -0,0 +1,182 @@
+    quantifiedAs:
+      qudt:unit: unit:Word
+      qudt:numericValue: 1
+    scalingFormula: M*Q
+    scalesPlan: True
+
+  - '@id': translators:9f112621-3f13-4f48-a3e2-fd2ca99493cd
+    '@type': RecipeFlow
+    action: accept
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    scalingFormula: Q
+
+  - '@id': translators:c7897c39-7f05-4a5d-a487-80e130a2414b
+    '@type': RecipeFlow
+    action: improve 
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    scalingFormula: Q
+
+  # resources
+
+  - '@id': translators:e1721a61-cd47-4556-84b9-8b1b81da15bf
+    '@type': EconomicResource
+    name: Summer of Commoning: A Tour of French Communities and Projects
+    classifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  # plan
+
+  - '@id': translators:02b39a30-3e04-4305-9656-7f261aa63c84
+    '@type': Process
+    name: Proofread Summer of Commoning
+    hasBeginning: 2018-12-07T08:00:00-5:00
+    hasEnd: 2018-12-09T08:00:00-5:0
+
+  - '@id': translators:a25500e0-0106-43cd-8cbb-e74779488835
+    '@type': Intent
+    skos:note: Proofreading a French to Spanish translation
+    action: work
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q834191 # proofreading
+    receiver: http://www.translators.example/
+    quantifiedAs:
+      qudt:unit: unit:Words
+      qudt:numericValue: 1025
+    after: 2018-12-07T08:00:00-5:00
+    before: 2018-12-09T08:00:00-5:00
+
+  - '@id': translators:a96bab2f-a1f0-47f3-827d-b4fb491d376f
+    '@type': Commitment
+    action: accept
+    provider: http://www.translators.example/
+    receiver: http://www.translators.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    after: 2018-12-07T08:00:00-5:00
+
+  - '@id': translators:f99ba192-fb8d-419e-8246-c8ebac531100
+    '@type': Commitment
+    action: improve
+    provider: http://www.translators.example/
+    receiver: http://www.translators.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q17928402 # blog post
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+    before: 2018-12-09T08:00:00-5:00
+
+  # commit to the work
+
+  - '@id': translators:856c43b1-0a63-445f-a56f-707b257f086d
+    '@type': Commitment
+    action: work
+    provider: http://www.alice.example/
+    receiver: http://www.translators.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q834191 # proofreading
+    quantifiedAs:
+      qudt:unit: unit:Words
+      qudt:numericValue: 1025
+    hasBeginning: 2018-12-08T10:00:00-5:00
+    hasEnd: 2018-12-08T15:00:00-5:00
```

Comment:
What if for scheduling purpose Alice wants to just include estimated duration of work but not select specific beginning and end yet, maybe just interval during which it happens like your *before* & *after* intent.

Reply:
Hmmm, commitments are I guess always estimates to some degree in terms of time.  Do you think hasBeginning and hasEnd cannot represent estimates?  I can see a lot of use for time:before in commitments, would work like "due".  Or are you meaning something like "6 hours"?  I don't think that helps much in terms of scheduling.  But I suppose whatever time: elements people want to use for scheduling they could do that, if they think that it will be supported by the receiving app.

Reply:
So you see no problem to have on commitment `time:hasDuration` plus some topological temporal relations like *before* and *after* ?

Reply:
I'm ok with that if it is not a quantity field in a different vocab and format.  If there is already a quantity field, and this is used for scheduling, not reciprocity etc.

Reply:
I think we agreed that in all UCs we can have variant where reciprocity gets handled based on %, so applications should make no assumptions if it gets handled based on quantity of involved resource, duration of flow or some other recorded quantity.

Reply:
Reciprocity is very different processing than scheduling, and reciprocity will be using various quantities to do its calculations - seems needlessly messy to ask that code to use different vocabularies for different quantities.

Since scheduling happens before the fact and is an estimate, perhaps that duration on a commitment is different that a time-based quantity on the event?

Reply:
> seems needlessly messy to ask that code to use different vocabularies for different quantities

I don't see any difference if terms come from one or many vocabs

> Since scheduling happens before the fact and is an estimate, perhaps that duration on a commitment is different that a time-based quantity on the event?

duration of the event fulfills duration of the commitment

Reply:
>duration of the event fulfills duration of the commitment

No it doesn't.  Fulfillment fulfills quantities.  Why complexify it?  Durations are for scheduling.

>I don't see any difference if terms come from one or many vocabs

So if you are using a quantity in an equation (such as happens with distribution logic), sometimes it is numericValue, sometimes it is numericDuration, and the units you might be comparing are a unit or a unitType?  Ouch.

Reply:
i don't think any case would involve comparing quantities of involved resources to duration of a flow.